Tests/ForageSDKTests/ForageServiceTests.swift

```diff
@@ -24,25 +24,26 @@ final class ForageServiceTests: XCTestCase {
         mockSession.data = forageMocks.tokenizeSuccess
         mockSession.response = forageMocks.mockSuccessResponse
         let service = LiveForageService(provider: Provider(mockSession))
-        
+
         let foragePANRequestModel = ForagePANRequestModel(
-            authorization: "authToken123",
-            merchantAccount: "merchantID123",
-            panNumber: "5076801234123412",
-            type: "ebt",
-            reusable: true
+           authorization: "authToken123",
+           merchantAccount: "merchantID123",
+           panNumber: "5076801234123412",
+           type: "ebt",
+           reusable: true,
+           customerID: "test-ios-customer-id"
         )
-        
+
         service.tokenizeEBTCard(request: foragePANRequestModel) { result in
-            switch result {
-            case .success(let response):
-                XCTAssertEqual(response.type, "ebt")
-                XCTAssertEqual(response.paymentMethodIdentifier, "d0c47b0ed5")
-                XCTAssertEqual(response.card.last4, "3412")
-                XCTAssertEqual(response.card.token, "tok_sandbox_72VEC9LasHbMYiiVWP9zms")
-            case .failure:
-                XCTFail("Expected success")
-            }
+           switch result {
+           case .success(let response):
+               XCTAssertEqual(response.type, "ebt")
+               XCTAssertEqual(response.paymentMethodIdentifier, "d0c47b0ed5")
+               XCTAssertEqual(response.card.last4, "3412")
+               XCTAssertEqual(response.card.token, "tok_sandbox_72VEC9LasHbMYiiVWP9zms")
```

Comment:
Might as well check for the `user_id` on the card here as well to validate the test.

Reply:
https://github.com/teamforage/forage-ios-sdk/pull/36/commits/29aaab9ff5660255f06872023d8b1962c446a869